pkg/web/router.go

```diff
@@ -124,9 +127,14 @@ func NewRouter(config *RouterConfig, wpool pool.WorkerPoolInterface) http.Handle
 		logger,
 	)
 
+	// apply authorization and validation middlewares using With to overcome issue with <id> URLParams not available
+	apiRouter := router.Group(func(r chi.Router) {
+		r.Use(authorizationMiddleware)
```

Comment:
let's validate this works correctly due to how route params get injected (that's why we were using `With`)